awesomerc.lua

```diff
@@ -459,8 +459,17 @@ end
 -- @DOC_CLIENT_BUTTONS@
 clientbuttons = gears.table.join(
     awful.button({ }, 1, function (c) client.focus = c; c:raise() end),
```

Comment:
Not important, but you could format the first binding like the two below.

Reply:
Yeah, I've been thinking about it, then I decided to follow this contributing guideline:

> do not try to fix style of code you are not writing, it's just adding noise for no gain;

Still, I can fix this one, provided that's fine with maintainers.

Reply:
Generally I'd agree, if one does lots of unrelated style fixes, it adds noise. Though in this case, it's related to the default config, which is mostly the only file that users will see and modify, so it should be as nice and consistent as possible IMO.

Reply:
Addressed in a9f871b.